runtime/doc/filetype.txt

```diff
@@ -521,6 +521,8 @@ To use Nvim as a manpager: >
 man.vim will always attempt to reuse the closest man window (above/left) but
 otherwise create a split.
 
+The case sensitivity of completion is controlled by 'fileignorecase'.
```

Comment:
Ah, good to know. Although I don't see `` `:foo` `` used very often. It's `|:foo|` as well. But yea, nothing we should care about.